### PR TITLE
New Rule - Do Not Use Meta Tags To Refresh or Redirect

### PR DESCRIPTION
**Pull request by tombui99**

### CODE REVIEW

File: rules/do-not-use-meta-tags-to-redirect/rule.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+</head>
+```
+**Figure: Bad Example that uses meta tags to refresh**
+
+``` html
+<head>
+  <meta name="description" content="...">
+</head>
+```
+**Figure: Good Example that does not use meta tags to refresh**
```
Comment (pierssinclairssw) [inline rules/do-not-use-meta-tags-to-redirect/rule.md:41]: I don't think we should copy/paste from another source. I recommend we reword it to fit within our own words.

Comment (tombui99) [reply rules/do-not-use-meta-tags-to-redirect/rule.md]: Done - reworded
